build.sbt

```diff
@@ -68,7 +81,7 @@ lazy val bsp4j = project
       "org.eclipse.lsp4j" % "org.eclipse.lsp4j.generator" % "0.5.0",
       "org.eclipse.lsp4j" % "org.eclipse.lsp4j.jsonrpc" % "0.5.0",
       "org.scala-sbt.ipcsocket" % "ipcsocket" % "1.0.0" % Test,
-      "ch.epfl.scala" % "bloop-frontend_2.12" % "1.0.0" % Test,
+      "ch.epfl.scala" % "bloop-frontend_2.12" % "46e63fc3" % Test,
```

Comment:
I'm investigating the reason why the CI is publishing these versions without the 1.0.0 prefix. As it's not critical, I propose to merge the PR with this version.